src/parser/parser_utils.cpp

```diff
@@ -110,21 +110,22 @@ void GetExpressionInfo(const expression::AbstractExpression* expr,
     case ExpressionType::STAR:
       inprint("*", num_indent);
       break;
-    case ExpressionType::COLUMN_REF:
-      // TODO: Fix this
-      inprint((expr)->GetExpressionName(), num_indent);
-      //if (expr->GetColumn() != NULL) inprint((expr)->GetColumn(), num_indent);
+    case ExpressionType::VALUE_TUPLE:
+      inprint((expr)->GetInfo().data(), num_indent);
```

Comment:
I think inprint is a nice helper function since it also handles the number of indents to give a nicer output? We may have to handle the indents manually if we use LOG_TRACE.